### ISSUE TITLE
Bookmark icon size is to small

### ISSUE THREAD

**Issue by oldman63** [koreader/koreader#5777]
* KOReader version: 2019.12
* Device: Kindle Paperwhite 1

If the page is bookmarked, the icon in up-right corner is very small (dogear.png 20x20 px).

How do increase the size of this icon?
Change for larger icon, not working. :-(

Please, put this icon size setting to menu.

**Comment by poire-z** [comment]
It should have a max of 1/32 of screen width - and can additionally be reduced depending on the margins you use (in cre/EPUB documents) - so as to not be drawn OVER the book text.
That's a design choice, that fixed a few issues. Details in #4081.

If you have small margins, try increasing them, and see if you get a bigger size.
The code if you feel like experimenting is:
https://github.com/koreader/koreader/blob/b111ccc2b803fc78425d9791fcc8bb7a9a704e6e/frontend/apps/reader/modules/readerdogear.lua#L11-L18
https://github.com/koreader/koreader/blob/b111ccc2b803fc78425d9791fcc8bb7a9a704e6e/frontend/apps/reader/modules/readerdogear.lua#L62-L75

I don't think it's worth having its size in the menu - but we may add a hidden/manual setting to either change the 1/32 or skip the cre margins restriction - depending on which of them bothers you.

But it has been like this for 18 months, any reason you're reacting just now (assuming you're a long time user as you're involved in translations) ? May be you have unusually set small margins? :)

**Comment by pazos** [comment]
I'm not sure why @oldman63 wants a bigger icon but if is just for figuring it which pages have a bookmark on a quick look I would recommend the "two finger right to left swipe", which will show you all the markers in the entire book.

**Comment by oldman63** [comment]
Thank you, for "Two finger right to left swype" tip!

But small bug found...

Tap upper-right corne -> bookmarked this page
Two finger right to left swype -> bookmarks
Long press on bookmark, Delete bookmark
Close Bookmarks page with [X] on upper-right
The bookmark icon always visible on the up-right corner!!! Don't removed.
After page left, Page right, the icon is disappeared.

**Comment by poire-z** [comment]
It's also available via multiswipe gestures, north-west by default - see picture in https://github.com/koreader/koreader/releases/tag/v2019.03.

Any answer to my above question about why your need for bigger icon? :)

**Comment by Frenzie** [comment]
There's also a wiki page: https://github.com/koreader/koreader/wiki/multiswipes

(Not entirely sure how you can reach it without searching for the page though.)

**Comment by AlanSP1** [comment]
@Frenzie all new articles are linked from [features list](https://github.com/koreader/koreader/wiki/Features-list), as this looked most logical place to link new articles. But I can link them from other places. Or at least mention that many options and articles are linked from features list.

What do you think?

**Comment by Frenzie** [comment]
Oh right, that's linked in the opening paragraph on the wiki. I added [this change](https://github.com/koreader/koreader/wiki/Home/_compare/986177a3ce05528fdd11dc81cd62b9ee73cac131...e9eb4f6e09ffcec6b473c4513befb6ae179798db) to repeat the information where it seems to fit in context.

It looks like a table of checkmarks rather than an index of available pages. I'm not entirely sure how much we can do about that within the confines of the GH wiki; can we put some kind of spoiler toggles in the sidebar for dropdowns maybe?

**Comment by AlanSP1** [comment]
I don't know what can be done with wiki here in GH, I only edit it on most essential level. If there's a way to do something like this, I don't know about it. Sorry.

**Comment by Frenzie** [comment]
@AlanSP1 I just investigated it a little bit, and what I was thinking could potentially work.

With this code:

```markdown
# [Tips and Tricks](https://github.com/koreader/koreader/wiki/Tips-and-Tricks)
<details><summary>Click for tips & tricks TOC</summary>

* [Getting started](https://github.com/koreader/koreader/wiki/Getting-Started)
* [Gestures](https://github.com/koreader/koreader/wiki/KOReader-Gestures)
* [Change defaults](https://github.com/koreader/koreader/wiki/Change-defaults)
* [DPI control](https://github.com/koreader/koreader/wiki/DPI-control)
* [Reflowing tweaks](https://github.com/koreader/koreader/wiki/Reflowing-tweaks)
* [Screenshots](https://github.com/koreader/koreader/wiki/KOReader-Screenshots)
* [Troubleshooting](https://github.com/koreader/koreader/wiki/Troubleshooting)
* [Android tips and tricks](https://github.com/koreader/koreader/wiki/Android-tips-and-tricks)
```

You get this result:
![Screenshot_2020-01-18_17-34-01](https://user-images.githubusercontent.com/202757/72667145-daa39080-3a18-11ea-8f07-4d7ede499d7f.png)

After clicking:
![Screenshot_2020-01-18_17-34-13](https://user-images.githubusercontent.com/202757/72667148-df684480-3a18-11ea-8ec8-d5e90a5fb518.png)

That specific example is awful since it's just a quick proof of concept. But it might work to group related things together in the sidebar, like different kind of gestures, multiswipes, gamepad, keymapping, …

We could also try numbering chapters or some such, perhaps? Like in https://github.com/ploopyco/mouse/wiki

Just brainstorming. Anyway, I'm happy to help too if something specific needs more attention. Just let us know. :-)

**Comment by AlanSP1** [comment]
OK, if you put first code, I could imitate (i.e. copy and paste) what you have done. And I need suggestions what should go in that section...

And, of course, thank you very much for your help. It is really needed and appreciated.

**Comment by Frenzie** [comment]
@AlanSP1 I've created a much better proof of concept that makes actual logical sense. The whitespace is a little bit finicky, it needs the slightly odd newline and such to look right in the result.

I think this concept shows some promise for making the sidebar more useful to find information, but if you have any concerns please share them.

<img src=https://user-images.githubusercontent.com/202757/72667432-476c5a00-3a1c-11ea-8de8-54a11d8599b9.png width=40%><img src=https://user-images.githubusercontent.com/202757/72667433-4804f080-3a1c-11ea-8c94-872652e6cdc8.png width=40%>

**Comment by AlanSP1** [comment]
Looks nice. If you will want something added there, mention it and I'll add it.

**Comment by Frenzie** [comment]
I think that specific list should be fine for now, but the rest of the sidebar could potentially be shortened, extended, and reorganized in similar ways.

For example under the `Usage` header I added (maybe better as `User interface`?) it could also have a 

* EPUB controls
  <details>

  * Style tweaks
  * alternate TOC
  * hyphenation
  * etc.
</details>

**Comment by AlanSP1** [comment]
I think Usage is better.

**Comment by poire-z** [comment]
Small bug at https://github.com/koreader/koreader/issues/5777#issuecomment-575703642 will be fixed by #5786.
Closing this issue as it looks like [@pazos tip](https://github.com/koreader/koreader/issues/5777#issuecomment-575653861) was enough for OP to be happy - and to not followup on answering my questions :)
